src/AriesSDK.js

```diff
@@ -99,6 +99,8 @@ const verifySelfie = (userId) => publishMessageToBusAndWaitForResponseWithMatchi
 
 const webViewLoaded = () => publishMessageToBusAndWaitForResponseWithMatchingId('view.loaded');
 
+const chatCreateCase = () => publishMessageToBusAndWaitForResponseWithMatchingId('chat.create-case');
```

Comment:
Is this line missing the case object arguments?

Reply:
indeed!! Good catch! thanks